module/zfs/vdev_file.c

```diff
@@ -254,6 +256,21 @@ vdev_ops_t vdev_file_ops = {
 	B_TRUE			/* leaf vdev */
 };
 
+void
+vdev_file_init(void)
+{
+	vdev_file_taskq = taskq_create("vdev_file_taskq", MAX(boot_ncpus, 16),
```

Comment:
It's a shame we can't easily use one of the many existing taskqs for this, but I don't see a nice way to do this either.  Alright, how about renaming the taskq to "z_vdev_file" for consistency with the other IO related taskqs.